### ISSUE TITLE
SchedulerStartOptions don't allow for 0

### ISSUE THREAD

**Issue by shakyShane** [kwonoj/rxjs-testscheduler-compat#38]
The following https://github.com/kwonoj/rxjs-testscheduler-compat/blob/master/src/TestScheduler.ts#L33-L41 will not allow a user to specify a `0` (zero) for created or subscribed values (as the zero will be seen as non-truthy

```js
if (!options.created) {
    ret.created = 100;
}
if (!options.subscribed) {
    ret.subscribed = 200;
}
if (!options.unsubscribed) {
    ret.unsubscribed = 1000;
}
```

**Comment by shakyShane** [comment]
Smallest possible example:

**Given the following**

```js
const scheduler = new TestScheduler();

const results = scheduler.startScheduler(() => {
    return Rx.Observable.of('hello world');
}, {created: 0, subscribed: 0, unsubscribed: 10});

console.log(results.messages.length);
```

**Expected**
I expect `results.messages` to contain 2 items, a next &  a complete

**Actual**
I see an error thrown `TypeError: Cannot read property 'unsubscribe' of undefined`

**Proposed solution**
Allow zero as created/subscribed value


Hope this helps, thank you  :)

**Comment by kwonoj** [comment]
This looks like my oversight. Let me try to look into it.

**Comment by kwonoj** [comment]
published v0.0.5.
